api-docs/api-reference/methods/post-save-instance-logs-version-accountid-instances-instanceid-action.rst

```diff
@@ -0,0 +1,125 @@
+
+.. _post-save-instance-logs-version-accountid-instances-instanceid-action:
+
+Save the DB logs for an instance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code::
+
+    POST /{version}/{accountId}/instances/{instanceId}/action
+
+Saves DB logs of the Instance.
+
+The logs will be saved in Cloud Files in the z_MYSQL_LOGS container.
```

Comment:
The logs will be saved in Cloud Files in the z_MYSQL_LOGS container. -> The logs are saved in Cloud Files in the z_MYSQL_LOGS container.

Using active voice to adhere to style guide found here https://developer.rackspace.com/docs/style-guide/how-to-article-guidelines/#active-voice